experiments/react-native-uifabric/src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,98 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View, Text } from 'react-native';
+import {
+  radioGroupName,
+  IRadioGroupType,
+  IRadioGroupProps,
+  IRadioGroupState,
+  IRadioGroupSlotProps,
+  IRadioGroupRenderData
+} from './RadioGroup.types';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { filterViewProps } from '../../utilities/RenderHelpers';
+import { settings } from './RadioGroup.settings';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { foregroundColorTokens, textTokens } from '../../tokens';
+import { useAsRadioGroupSelection } from '../../hooks';
+
+// Creates context to allow communication between its children components (RadioButtons)
+export const RadioGroupContext = React.createContext({
```

Comment:
You can add a type here. `IRadioGroupContext` or something and pass it in as a generic to createContext.

Reply:
Changed